### ISSUE TITLE
[editorial] The (WCAG 2.1) Understanding 2.4.7 doc is mentioning SC 2.4.11 of WCAG 2.2

### ISSUE THREAD

**Issue by Malvoz** [w3c/wcag#1085]
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html:

> Note that a keyboard focus indicator can take different forms. This criterion does not specify what the form is, but [Focus Visible (Enhanced)](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html#) does define how visible the indicator should be. Passing [Focus Visible (Enhanced)](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html#) would pass this success criterion.

To my understanding this shouldn't refer to WCAG 2.2, correct? (The links are broken either way.)

**Comment by guyhickling** [comment]
I agree with @Malvoz. SC2.4.11 could refer back to 2.4.7 in order to make the above comment, and that should be sufficient to make the point. Nothing in the WCAG 2.1 should refer to 2.2!

**Comment by Malvoz** [comment]
The content seems to have changed, this is no longer relevant.
